transformers/run_data2vec.sh

```diff
@@ -86,7 +86,7 @@ do
 
     python run_eval.py \
         --model_id=${MODEL_ID} \
-        --dataset_path="open-asr-leaderboard/datasets-test-only" \
+        --dataset_path="hf-audio/esb-datasets-test-only-sorted" \
```

Comment:
@sanchit-gandhi  Were you able to valid sorted data? I noticed few files are corrupted and not matching the reference transcript. Also Total number of files from previous unsorted to this set is not matching.